README.md

```diff
@@ -18,103 +19,117 @@ It's implemented as a k8s operator, an active controller component which acts up
 * Backlog: [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2192232)
 * Docker: https://hub.docker.com/r/cfcontainerization/cf-operator/tags
 
-
 ## Requirements
 
-A working kubernetes cluster.
+- A working Kubernetes cluster
+- Helm binary
+- Go 1.12.2 and install the tool chain: `make tools`
 
-## Install
 
-The alpha release of cf-operator has to be built from source. The helm chart is not working yet.
+## Dependencies
 
-    git checkout v0.1.0
+Run with libraries fetched via go modules:
 
-We need to build the binary, which can run out of cluster:
+```bash
+export GO111MODULE=on
+```
 
-    bin/build
+Or with a vendor folder, using GO111MODULE=off, this also speeds up docker builds
```

Comment:
Could we only provide one solution for using `vendor/`? Too many options give user puzzled.

Reply:
+1 , good point, lets stick with go modules